app/jobs/send_patient_otp_sms_job.rb

```diff
@@ -0,0 +1,14 @@
+class SendPatientOtpSmsJob < ApplicationJob
+  queue_as :default
+  self.queue_adapter = :sidekiq
+
+  def perform(patient_authentication)
```

Comment:
Typically you want to pass model IDs to jobs and do a `find`. Passing them like this serializes the object in Redis. For this one it's fine for now, just a heads up.

Reply:
From Rails 5, ActiveJob now handles ActiveRecord objects seamlessly using GlobalID ([check it out](https://guides.rubyonrails.org/v5.0/active_job_basics.html#globalid)), so we should be totally fine to pass in the record directly!

Reply:
Nice! TIL